libfreerdp/utils/test/TestFuzzUtilsReadPassphrase.c

```diff
@@ -2,6 +2,6 @@
 
 int LLVMFuzzerTestOneInput(const uint8_t* Data, size_t Size)
 {
-	freerdp_passphrase_read(NULL, (char*)Data, Size, 0);
```

Comment:
you misunderstood, this function can not be tested like this.
No matter which argument you choose, it will wait for external input...
What you provide is the buffer the data will be read to.

Reply:
removed it at all